src/Files.App/Views/MainPage.xaml.cs

```diff
@@ -175,9 +177,13 @@ public void MultitaskingControl_CurrentInstanceChanged(object? sender, CurrentIn
 				SidebarAdaptiveViewModel.PaneHolder.PropertyChanged -= PaneHolder_PropertyChanged;
 
 			var navArgs = e.CurrentInstance.TabItemParameter?.NavigationParameter;
-			SidebarAdaptiveViewModel.PaneHolder = e.CurrentInstance as IPaneHolder;
+			if (e.CurrentInstance is not IPaneHolder currentInstance || navArgs is null) 
+   				return;
+			SidebarAdaptiveViewModel.PaneHolder = currentInstance;
 			SidebarAdaptiveViewModel.PaneHolder.PropertyChanged += PaneHolder_PropertyChanged;
-			SidebarAdaptiveViewModel.NotifyInstanceRelatedPropertiesChanged((navArgs as PaneNavigationArguments).LeftPaneNavPathParam);
+			if (navArgs is not PaneNavigationArguments pageNav)
+   				return;
+			SidebarAdaptiveViewModel.NotifyInstanceRelatedPropertiesChanged(pageNav.LeftPaneNavPathParam);
```

Comment:
```suggestion
			SidebarAdaptiveViewModel.NotifyInstanceRelatedPropertiesChanged((navArgs as PaneNavigationArguments)?.LeftPaneNavPathParam);
```
Here, too, it is better to let the method continue as much as possible than to exit it.

Reply:
Good point, the argument is nullable